portfolio/src/main/webapp/requests/queries/index.js

```diff
@@ -0,0 +1,24 @@
+const get_comments = async () => {
+    const payload = await fetch('/comments');
+    const comments = await payload.json();
+
+    const comments_section = document.getElementById("comments");
+    comments.forEach(comment_payload => {
+        const username = document.createElement("span");
+        const comment = document.createElement("p");
+
+        username.innerHTML = `@ ${comment_payload.username}`;
+        username.classList.add("badge", "badge-dark");
+        comment.innerHTML = comment_payload.comment;
+        
+        const comment_to_append = document.createElement("li");
```

Comment:
nit: the Google Javascript Style Guide [suggests](https://google.github.io/styleguide/jsguide.html#naming-local-variable-names) that local variable names should be lowerCamelCased, i.e. `commentToAppend`